app/controllers/photos_controller.rb

```diff
@@ -5,7 +5,10 @@ class PhotosController < ApplicationController
   # GET /photos
   # GET /photos.json
   def index
-    @photos = @album.photos
+    @desc = true
+    @desc = false if params[:desc] == 'true'
```

Comment:
You don't seem to be using the @desc instance variable anywhere. Did you mean to pass this to the order_by method?